client/curriculum/html/HTMLStructure/HTMLStructure.tsx

```diff
@@ -12,6 +12,7 @@ import { getLessonById } from '~/services/Constants';
 import Highlight from '~/components/Highlight/Highlight';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.